run_spider.py

```diff
@@ -0,0 +1,66 @@
+from scrapy.crawler import CrawlerProcess
+from scrapy.utils.project import get_project_settings
+from website_bodytext_scraper.spiders.detect_stripe import DetectStripeSpider
+import asyncio
+import nest_asyncio
+import sys
+import logging
+import os
+
```

Comment:
To separate the settings per environment, I propose setting an environment variable, could be called "SCRAPER_ENV" for example. In the settings.py file (which I think is being loaded when get_project_settings is called?), you can write some logic to get the value of SCRAPER_ENV and assign local or Databricks params as needed